pubsub/kafkapubsub/kafka_test.go

```diff
@@ -374,6 +385,7 @@ func TestMultiplePartionsWithRebalancing(t *testing.T) {
 	}
 	defer func() {
 		if err := sub2.Shutdown(ctx); err != nil {
+			fmt.Printf("shutting down sub2\n")
```

Comment:
Do you want this? Or left over from debugging?

Reply:
Left over; removed.